base/applications/charmap/lang/he-IL.rc

```diff
@@ -1,6 +1,14 @@
 /* Translated by Baruch Rutman */
 LANGUAGE LANG_HEBREW, SUBLANG_DEFAULT
 
+/*
+ * Enable RTL mirroring by adding two left-to-right marks (LRMs)
+ * in front of the FileDescription value of the version stamping.
+ */
+#undef REACTOS_STR_FILE_DESCRIPTION
```

Comment:
I think undefining `REACTOS_STR_FILE_DESCRIPTION` is an improper manner.
We can change `<reactos/version.rc>`.

Reply:
And you would change it to what?

Reply:
```
VALUE "FileDescription",  L"\x200e\x200e"  REACTOS_STR_FILE_DESCRIPTION
```

Reply:
But version.rc is the template file also used for non RTL languages. That's why you don't want to do that change always.

One way around, could be to keep the `REACTOS_STR_FILE_DESCRIPTION` define, and for RTL languages (in their resource file), define a `REACTOS_STR_FILE_DESCRIPTION_RTL`, or maybe some other macro name (e.g. `REACTOS_RTL_RESOURCE`), and in the version.rc template, do something along the lines of:
```
#ifdef REACTOS_RTL_RESOURCE
VALUE "FileDescription",  L"\x200e\x200e"  REACTOS_STR_FILE_DESCRIPTION
#undef REACTOS_RTL_RESOURCE
#else
VALUE "FileDescription",  REACTOS_STR_FILE_DESCRIPTION
#endif
```
Also, note that having unicode L string tag added in front of the description string would be mandatory. With the current (or yours) proposition, it is not, and that would break compilation with some compilers.

Reply:
> In any case, one would have to re-include version.rc in the language resource file...

Why? Reason?

Reply:
Define `REACTOS_RTL_RESOURCE` in Hebrew or Arabib language files, and later include `version.rc`.

Reply:
The reason is that the first inclusion of version.rc done in the main resource file, creates the language-neutral version resource. To get the mechanism of this PR to work, one would also need a language-specific version resource, and the only way to do that is to re-include version.rc from within the localized (per-language) resource file, e.g. he-IL.rc , so that this lang-specific version resource gets the localized file version description string containing the two left-to-right symbols.

Reply:
> need a language-specific version resource

I see. Thanks.

Reply:
If we want some language-specific version info, then some properties might be changed:

https://github.com/reactos/reactos/blob/139809b747aed5f0afc34734d5703845efbcd435/sdk/include/reactos/version.rc#L118

https://github.com/reactos/reactos/blob/139809b747aed5f0afc34734d5703845efbcd435/sdk/include/reactos/version.rc#L138